test/HistorySuggestion.cpp

```diff
@@ -0,0 +1,40 @@
+#include "../src/HistorySuggestion.hpp"
+#include "helpers/Helpers.hpp"
+#include "catch.hpp"
+
+using namespace yb;
+
+namespace {
+
+void testSuggestions(HistorySuggestion &suggestion, std::string chars, std::initializer_list<std::string> expected) {
+    for (std::pair<std::string::const_iterator, std::initializer_list<std::string>::const_iterator> it{chars.begin(), expected.begin()};
```

Comment:
I iterate through 2 containers, and `auto` gives me this: `error: initializer for variable 'it' with type 'auto' contains multiple expressions`